net.lugsole.bible_gui.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id" : "net.lugsole.bible_gui",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "bible",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "libhandy",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": [
+                "-Dglade_catalog=disabled"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://source.puri.sm/Librem5/libhandy.git",
+                    "commit":"7a193d7692c9c76a1a94f17c4d30b585f77d177c"
+                }
+            ]
+        },
+        {
+            "name": "espeak",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/espeak-ng/espeak-ng",
+                    "commit": "b702b03996de94035fadae0eb5ad9506c5a09f35"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+            	"printenv",
+            	"pwd",
+            	"./autogen.sh",
+            	"./configure --bindir=/app/bin --libdir=/app/lib --prefix=/app/",
+                "make all",
+                "make install"
```

Comment:
Nitpick: split those into seperate lines.

Reply:
Not a nitpick anymore. This builds just fine without specifying the build-commands and using autotools (it's default one) as a build system